### ISSUE TITLE
Bumper Ring Speed adjustment bug.

### ISSUE THREAD

**Issue by wrd1972** [vpinball/vpinball#209]
**Describe the bug**
I am trying to use the "Ring Speed" adjustment for a bumper object on an EM table, and am having no luck. I have tried to use this adjustment to slow the ring speed, but it does not seem to work. Regardless of the value, the animation speed seems to remain the same. 

**To Reproduce**

**Expected behavior**
I asm expecting to be able to vary the ring drop animation as needed.

**Screenshots**

**Crash dump**
If it's about a crash, please attach `crash.dmp` as well.

**Versions**
 - VPX: [e.g. 1.7.0 Final Revision 366 32 bit - see *Help - About*]
 - Windows version 10
 - GPU and video settings if applicable

**Additional context**

**Comment by toxieainc** [comment]
If i remember correctly @fuzzelhjb did implement this back then..

**Comment by vbousquet** [comment]
I have just tested this on latest 10.8 (after the animation implementation changes, v643) and it worked as expected.
This can be checked with the following steps:
- Run VPX, Create the full example table
- Run the table
- In a debug view, run the following code: ` Bumper1.RingSpeed = 0.01: Bumper1.PlayHit`

The ring will move down and up very slowly. I will close the issue. If I messed up and it's another bug, we will reopen it
